NEWS.md

```diff
@@ -2,6 +2,10 @@
 
 # data.table [v1.15.99](https://github.com/Rdatatable/data.table/milestone/30)  (in development)
 
+## POTENTIALLY BREAKING CHANGES
+
+1. In `DT[, j := value]`, when value is class `POSIXlt`, we automatically coerce it to class `POSIXct` instead, [#1724](https://github.com/Rdatatable/data.table/issues/1724). Thanks to @linzhp for the report, and Benjamin Schwendinger for the fix.
```

Comment:
this PR looks great overall, but I would change `DT[, j := value]` to `DT[, variable := value]` to avoid confusion (j is both the variable name and the argument name)
```r
> data.table()[j= variable := 5][]
   variable
      <num>
1:        5
```

Reply:
good point!